src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -384,21 +390,21 @@ class LedgerMasterImp
 
         // VFALCO TODO recreate the CanonicalTxSet object instead of resetting it
         mHeldTransactions.reset (engine.getLedger()->getHash ());
-        mCurrentLedger.set (engine.getLedger ());
+        mCurrentLedger.set(engine.getLedger());
     }
 
-    LedgerIndex getBuildingLedger ()
+    LedgerIndex getBuildingLedger () override
     {
         // The ledger we are currently building, 0 of none
         return mBuildingLedgerSeq.load ();
     }
 
-    void setBuildingLedger (LedgerIndex i)
+    void setBuildingLedger (LedgerIndex i) override
     {
         mBuildingLedgerSeq.store (i);
     }
 
-    TER doTransaction (STTx::ref txn, TransactionEngineParams params, bool& didApply)
+    TER doTransaction (STTx::ref txn, TransactionEngineParams params, bool& didApply) override
     {
         Ledger::pointer ledger;
         TransactionEngine engine;
```

Comment:
I'll suggest that `result` on line 411 be initialized with `telFAILED_PROCESSING` or some such.  That will make it easier to catch possible problems with future refactoring.